src/services/services.ts

```diff
@@ -1822,6 +1822,13 @@ namespace ts {
                 return false;
             }
 
+            switch (openingBrace) {
+                case CharacterCodes.singleQuote:

```

Comment:
Indeed. I've never liked any completions when typing inside a comment (including braces). Is there a rationale for what we do provide and what we don't?